contracts/truefi2/FixedTermLoanAgency.sol

```diff
@@ -301,32 +312,41 @@ contract FixedTermLoanAgency is IFixedTermLoanAgency, UpgradeableClaimable {
      * @param loanToken LoanToken to fund
      */
     function fund(ILoanToken2 loanToken) external {
-        require(msg.sender == loanToken.borrower(), "FixedTermLoanAgency: Sender is not borrower");
-        ITrueFiPool2 pool = loanToken.pool();
+        // TODO Replace hardcoded _maxApy value with a param
+        uint256 _maxApy = 10000;
 
+        ITrueFiPool2 pool = loanToken.pool();
         require(factory.isSupportedPool(pool), "FixedTermLoanAgency: Pool not supported by the factory");
```

Comment:
Don't know if you are debugging this right know, but it seems tests are failing because `factory` is not set.

Reply:
Thanks! Yeah, I'm currently debugging this and working on a fix.

Reply:
I just merged in your FTLA deletion changes, and that seems to have fixed one test, but I'm still not sure how to fix the others.

Reply:
Could someone else take a look at and debug these test failures?

I have a sense for the general cause (mock contracts not set properly), but I've spent several hours and haven't tracked down exactly which lines need to change, or to what.